test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/KeyValuePairModelBinderTest.cs

```diff
@@ -231,7 +232,8 @@ private static ModelBindingContext CreateContext()
                     MetadataProvider = metataProvider,
                     ValidatorProvider = new DataAnnotationsModelValidatorProvider(
                         new TestOptionsManager<MvcDataAnnotationsLocalizationOptions>(),
-                        stringLocalizerFactory: null)
+                        stringLocalizerFactory: null,
+                        validationAttributeAdapterProvider: new ValidationAttributeAdapterProvider())
```

Comment:
The updated tests look rather strange with the parameter order you've chosen. Normal practise in this repo is to put the optional arguments last. That will also remove the need to name the `validationAttributeAdapterProvider` arguments.